data/json/snippets/survivor_notes.json

```
@@ -687,6 +687,169 @@
+      },
+      {
+        "id": "msn_note_08",
+        "text": "\"…I believe these things can't see well at night either.  If you're quiet you might just pass a horde.\""
+      },
+      { "id": "msn_note_09", "text": "\"I doubt they can smell anything if you light some tires on fire.\"" },
+      {
+        "id": "msn_note_10",
+        "text": "\"Remember when I told you about that moose coming right at me last year?  Try shouting at them.\""
+      },
+      {
+        "id": "msn_note_11",
+        "text": "\"Look, I know they look like kids, but you gotta take 'em out.  You'll get used to it, just like I have.  Remember, those things aren't human anymore.\""
+      },
+      {
+        "id": "msn_note_12",
+        "text": "\"Always plan ahead, never sleep in a place that has only one exit.  Lost a good friend to a bad decision.\""
+      },
+      {
+        "id": "msn_note_13",
+        "text": "\"If you're planning on using a building for a temporary stash, might be a good idea to open and close the windows.  Sure, having unlocked windows might seem unsafe, but if I'm trying to sneak back in while some Zed's are wandering near the door, you'll thank me later for the quiet entrance.\""
+      },
+      {
+        "id": "msn_note_14",
+        "text": "\"Mama always used to say life was like a box of chocolates, sometimes it gives you radiation poisoning.\""
+      },
+      {
+        "id": "msn_note_15",
+        "text": "\"I don't know if we each have a destiny, or if we're all just floatin' around accidental-like on a breeze, but I, I think maybe it's both.  Maybe both is happenin' at the same time.  I mid'ha ate too many of them hallucina'gan mushrooms.\""
+      },
+      {
+        "id": "msn_note_16",
+        "//": "Thanks to Lorith for helping me by providing this snippet idea.",
+        "text": "\"I swear I am not drunk, but I just saw a bear riding a motorcycle.\""
+      },
+      {
+        "id": "msn_note_17",
+        "//": "Thanks to Lorith for helping me by providing this snippet idea.",
+        "text": "\"Heard noises outside.  Saw giant scrapheap move by.  Not going out, nope, never.\""
+      },
+      {
+        "id": "msn_note_18",
+        "//": "Thanks to Lorith for helping me by providing this snippet idea.",
+        "text": "\"I thought it was a zombie, but there was this thing in the distance, looked like nothing but metal parts stuck together.  A zombie attacked it and it spewed lightning all over, then sprouted blades and minced it.  Not going out, nuh uh.\""
+      },
+      {
+        "id": "msn_note_20",
+        "//": "Thanks to Vetulus for helping me by providing this snippet idea.",
+        "text": "\"Why are their so many fucking giant spiders?\""
+      },
+      {
+        "id": "msn_note_21",
+        "//": "Thanks to Vetulus for helping me by providing this snippet idea.",
+        "text": "\"Partake of the fruit, eat the seed, accept the sap.\""
```

```suggestion
        "text": "\"Partake of the fruit, eat the seed, accept the sap.\""
```
I thought this was one of the few things we kept spoiler free in the game?

---

Although it describes the process for Mycus mutation, it doesn't reference the Mycus directly in any fashion. Stumbling on this by itself, you wouldn't know what it was talking about.